ips-ag/demo-ab-testing-azure
# Feature/65971928 playground project initialization

## Summary
This PR initializes the playground project with a .NET Core backend and an Angular frontend, setting up the basic structure and essential configurations.

## Changes
- Initialized .NET Core API project with essential configurations.
- Set up Angular project with HTTP communication and UI libraries.
- Added README with setup and contribution guidelines.

## Testing
Verified project setup across environments and ensured compatibility.

## How to Test
1. Clone the repo.
2. Follow README setup instructions.
3. Verify initialization without errors.
